docs/index.md

```diff
@@ -17,5 +17,14 @@ This document specifies the future protocol for diaspora. diaspora\* release 0.6
 [entities][entities] with this protocol, but still sends entities with an older protocol. Starting with diaspora\*
 release X this protocol is fully supported.
 
+## Implementations
+
+An implementation of this protocol is available as a Ruby Gem under the AGPL [on Github][github]. This is the library used by the diaspora* project.
+
+The [Friendica][friendica] project also has [its implementation in PHP][phpimplementation].
+
 [communication]: https://wiki.diasporafoundation.org/How_we_communicate
 [entities]: {{ site.baseurl }}/entities/
```

Comment:
This document doesn't get rendered well at the github MD viewer:
![2017-01-20 03-57-46](https://cloud.githubusercontent.com/assets/10187586/22132756/a791a6b2-dec4-11e6-9cdf-55c65b08e450.png)

(Look at the bottom line)